infra/admin.bicep

```
@@ -12,6 +12,7 @@ param postgresUser string
 param postgresPassword string
 param postgresDatabase string
 param postgresServer string
+param postgresEncryptionKey string
```

```suggestion
@secure()
param postgresEncryptionKey string
```

Probably should mark this as secure here too